packages/widget/src/components/internal/dropdown/styles.ts

```diff
@@ -0,0 +1,107 @@
+import { css } from 'lit';
+
+export const styles = css`
+  .dropdownWrapper {
+    min-width: 7.5rem;
+    position: relative;
+    width: 100%;
+    height: 100%;
+  }
+
+  .dropdown {
+    height: 100%;
+  }
+
+  .dropdownTrigger {
+    height: 100%;
+    width: 100%;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    cursor: pointer;
+    padding: 0.25rem 0;
+    box-sizing: border-box;
+
+    .optionName {
+      max-width: 4rem;
+    }
+
+    &.disabled {
+      cursor: not-allowed;
+      opacity: 0.5;
+    }
+  }
+
+  .chevron {
+    display: flex;
+    align-items: center;
+    transform: rotate(0deg);
+    transition: transform 0.3s ease;
+
+    &.open {
+      transform: rotate(180deg);
+    }
+  }
+
+  .dropdownContent {
+    display: none;
+    position: absolute;
+    background-color: white;
```

Comment:
I think we have a color var for this if not mistaken unless we need this to be actually white